x-pack/plugins/observability_solution/observability_ai_assistant/public/components/message_panel/message_text.tsx

```diff
@@ -156,7 +156,14 @@ export function MessageText({ loading, content, onActionClick }: Props) {
       },
       table: (props) => (
         <>
-          <EuiTable {...props} />
+          <EuiTable
+            {...props}
+            className={css`
+              .euiTableCellContent__text {
+                white-space: normal;
+              }
+            `}
```

Comment:
nit: Feel hacky/fragile to override EUI this way. Could easily break in an update (eg. Borealis theme)

Reply:
as in, the className for `euiTableCellContent__text` would change?